<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>861</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(87, 126, 255);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Employé</string>
     </attribute>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>791</width>
        <height>371</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QPushButton" name="pushButton_30">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>0</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>TRI ID</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_33">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>0</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>TRI SALAIRE</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_afficher">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>320</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Afficher</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_31">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Statistique</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="rechercher">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>0</y>
          <width>141</width>
          <height>24</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Recherche</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QTableView" name="tab_emp">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>721</width>
          <height>111</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>170</y>
          <width>731</width>
          <height>141</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(90, 170, 72);</string>
        </property>
        <property name="title">
         <string>Employé</string>
        </property>
        <widget class="QTabWidget" name="tabWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>-1</y>
           <width>731</width>
           <height>141</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Ajouter Employé</string>
          </attribute>
          <widget class="QLabel" name="label_11">
           <property name="geometry">
            <rect>
             <x>220</x>
             <y>20</y>
             <width>47</width>
             <height>14</height>
            </rect>
           </property>
           <property name="text">
            <string>Prénom :</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_14">
           <property name="geometry">
            <rect>
             <x>440</x>
             <y>60</y>
             <width>47</width>
             <height>14</height>
            </rect>
           </property>
           <property name="text">
            <string>Salaire :</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_13">
           <property name="geometry">
            <rect>
             <x>440</x>
             <y>20</y>
             <width>47</width>
             <height>14</height>
            </rect>
           </property>
           <property name="text">
            <string>Agence :</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_10">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>70</y>
             <width>41</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Nom :</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="email">
           <property name="geometry">
            <rect>
             <x>510</x>
             <y>90</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="salaire">
           <property name="geometry">
            <rect>
             <x>510</x>
             <y>60</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_15">
           <property name="geometry">
            <rect>
             <x>440</x>
             <y>90</y>
             <width>47</width>
             <height>14</height>
            </rect>
           </property>
           <property name="text">
            <string>Email</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="prenom">
           <property name="geometry">
            <rect>
             <x>280</x>
             <y>20</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_16">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>20</y>
             <width>41</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>ID :</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="nom">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>70</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_12">
           <property name="geometry">
            <rect>
             <x>220</x>
             <y>50</y>
             <width>47</width>
             <height>14</height>
            </rect>
           </property>
           <property name="text">
            <string>Adresse :</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="id">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>20</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="ref_agence">
           <property name="geometry">
            <rect>
             <x>510</x>
             <y>20</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QComboBox" name="adresse">
           <property name="geometry">
            <rect>
             <x>280</x>
             <y>50</y>
             <width>111</width>
             <height>22</height>
            </rect>
           </property>
           <item>
            <property name="text">
             <string>Bizerte</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Tunis</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Nabeul</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sfax</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sousse</string>
            </property>
           </item>
          </widget>
          <widget class="QLabel" name="label_24">
           <property name="geometry">
            <rect>
             <x>230</x>
             <y>90</y>
             <width>41</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>PWD</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="mdp">
           <property name="geometry">
            <rect>
             <x>280</x>
             <y>90</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Modifier Employé</string>
          </attribute>
          <widget class="QPushButton" name="pushButton_modifier">
           <property name="geometry">
            <rect>
             <x>600</x>
             <y>90</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Modifier</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_17">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>60</y>
             <width>41</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Nom :</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="agence_modif">
           <property name="geometry">
            <rect>
             <x>480</x>
             <y>10</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_18">
           <property name="geometry">
            <rect>
             <x>410</x>
             <y>50</y>
             <width>47</width>
             <height>14</height>
            </rect>
           </property>
           <property name="text">
            <string>Salaire :</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_19">
           <property name="geometry">
            <rect>
             <x>410</x>
             <y>80</y>
             <width>47</width>
             <height>14</height>
            </rect>
           </property>
           <property name="text">
            <string>Email</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="prenom_modif">
           <property name="geometry">
            <rect>
             <x>250</x>
             <y>10</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_20">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>60</y>
             <width>47</width>
             <height>14</height>
            </rect>
           </property>
           <property name="text">
            <string>Adresse :</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_21">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>41</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>ID :</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="id_modif">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>10</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="nom_modif">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>60</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_22">
           <property name="geometry">
            <rect>
             <x>410</x>
             <y>10</y>
             <width>47</width>
             <height>14</height>
            </rect>
           </property>
           <property name="text">
            <string>Agence :</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="email_modif">
           <property name="geometry">
            <rect>
             <x>480</x>
             <y>80</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_23">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>10</y>
             <width>47</width>
             <height>14</height>
            </rect>
           </property>
           <property name="text">
            <string>Prénom :</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="salaire_modif">
           <property name="geometry">
            <rect>
             <x>480</x>
             <y>50</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QComboBox" name="adresse_modif">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>60</y>
             <width>111</width>
             <height>22</height>
            </rect>
           </property>
           <item>
            <property name="text">
             <string>Bizerte</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Tunis</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Nabeul</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sfax</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sousse</string>
            </property>
           </item>
          </widget>
          <widget class="QLabel" name="label_25">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>90</y>
             <width>41</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>PWD</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="mdp_2">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>90</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButton_34">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>0</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>stat salaire</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_supprimer">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>320</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_35">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>0</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>TRI NOM</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="id_supp">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>320</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="ajouter">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>320</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_29">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>0</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>fichier pdf</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_32">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>0</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>QR code</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>20</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background: rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string>ImmoClick</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_26">
        <property name="geometry">
         <rect>
          <x>205</x>
          <y>150</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background: rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string>Mot de passe</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_27">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>100</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background: rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string>EMAIL :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="mdp_login">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>150</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background: rgb(255, 255, 255)</string>
        </property>
       </widget>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>481</width>
          <height>251</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background: rgb(255, 255, 255)</string>
        </property>
        <widget class="QPushButton" name="login">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>180</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>LOGIN</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="email_login">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>90</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: rgb(255, 255, 255)</string>
         </property>
        </widget>
       </widget>
       <zorder>widget_2</zorder>
       <zorder>label_3</zorder>
       <zorder>label_26</zorder>
       <zorder>label_27</zorder>
       <zorder>mdp_login</zorder>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>Clients</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>340</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>340</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>340</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Agences</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Biens</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Réclamations</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>934</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
